webapp/public/locales/fr/main.json

```diff
@@ -467,11 +468,14 @@
   "study.modelization.clusters.thermal.op4": "Autre polluant 4 (t/MWh)",
   "study.modelization.clusters.thermal.op5": "Autre polluant 5 (t/MWh)",
   "study.modelization.clusters.operatingCosts": "Coûts d'exploitation",
-  "study.modelization.clusters.marginalCost": "Coûts marginaux (€/MWh)",
-  "study.modelization.clusters.fixedCost": "Coûts fixes (€/h)",
+  "study.modelization.clusters.marginalCost": "Coût marginal (€/MWh)",
+  "study.modelization.clusters.fixedCost": "Coûts fixes O&M (€/h)",
   "study.modelization.clusters.startupCost": "Coûts de démarrage (€)",
   "study.modelization.clusters.marketBidCost": "Offre de marché (€/MWh)",
-  "study.modelization.clusters.spreadCost": "Spread (€/MWh)",
+  "study.modelization.clusters.spreadCost": "Random Spread (€/MWh)",
+  "study.modelization.clusters.variableOMCost": "Coût variable O&M (€/MWh)",
+  "study.modelization.clusters.costGeneration": "TS Cost",
```

Comment:
Missing translations for `spreadCost` and `costGeneration`

Reply:
for `spreadCost` it's not missing the field is the same it's intended. For `costGeneration` it has to be confirmed and added for fr key if needed (not provided)